virtcontainers/device/config/config.go

```diff
@@ -90,11 +90,49 @@ type DeviceInfo struct {
 // VhostUserDeviceAttrs represents data shared by most vhost-user devices
 type VhostUserDeviceAttrs struct {
 	DevType    DeviceType
-	DeviceInfo DeviceInfo
+	DeviceInfo *DeviceInfo
 	SocketPath string
 	ID         string
 }
 
+// BlockDrive represents a block storage drive which may be used in case the storage
+// driver has an underlying block storage device.
+type BlockDrive struct {
+
+	// Path to the disk-image/device which will be used with this drive
```

Comment:
comment should start with `File`, isn't it?

Reply:
Correct. I'll fix it, thanks!